### ISSUE TITLE
Window length in bz_FindRipples.m

### ISSUE THREAD

**Issue by 37ig** [buzsakilab/buzcode#418]
Hi, 

I'm a bit confused about how the RMS window length is computed in bz_FindRipples.m

Lines 132 reads: windowLength = frequency/frequency*11;

Which should always set the windowLength to 11 data-points. Is this as intended or an error?

Thanks for any help!

**Comment by brendonw1** [comment]
That is strange.  That line is very odd and may even always result in a
value of 11 (? depending on order of operations).
I am cc'ing Antonio and David in case they might know more.  I've almost
never used that code and definitely didn't significantly write any of it

Brendon

Brendon Watson, MD-PhD
Assistant Professor in Psychiatry
Biomedical Sciences Research Building, Room 5059
University of Michigan
109 Zina Pitcher Place
Ann Arbor, MI 48109-5720
Lab Website: http://watsonneurolab.org
Clinical phone: 734-764-0231
<http://voice.google.com/calls?a=nc,%2B17347640231>

**Comment by 37ig** [comment]
Yes the order of operations means it does return 11. With a bit more investigation it looks like the bz_FindRipples.m may have been inspired by (or vice versa) the FMAToolbox function FindRipples.m.

The line in the FMAToolbox function is 
windowLength = round(frequency/1250*11);
which makes much more sense. But would be useful if someone could confirm this please. 

Cheers!

**Comment by dlevenstein** [comment]
This should absolutely be 

windowLength = round(frequency/1250*11);

not sure when it got changed, or who's using that code rn - anyone?

**Comment by dlevenstein** [comment]
actually - I bet what happened is someone wanted to change the 1250 to be the sampling frequency from the sessionInfo, and then got messed up between the frequency of the filter vs the sampling frequency....

**Comment by evilrobotxoxo** [comment]
IMO this should be written as

round(11 * frequency/1250)

because under PEMDAS frequency/1250*11 is equal to frequency/(1250*11), but
under BODMAS it's equal to (frequency/1250)*11

We should avoid that sort of ambiguity.
